go-controller/pkg/ovn/master.go

```diff
@@ -127,15 +127,14 @@ func (oc *Controller) Start(kClient kubernetes.Interface, nodeName string) error
 func (oc *Controller) StartClusterMaster(masterNodeName string) error {
 	// The gateway router need to be connected to the distributed router via a per-node join switch.
 	// We need a subnet allocator that allocates subnet for this per-node join switch. Use the 100.64.0.0/16
-	// or fd98::/64 network range with host bits set to 3. The subnetallocator will allocate a subnet that
-	// has up to 6 host IPs)
+	// or fd98::/64 network range with a hostSubnetLength allowing 3 bits of host IPs.
 	if config.IPv4Mode {
 		_, joinSubnetCIDR, _ := net.ParseCIDR(config.V4JoinSubnet)
-		_ = oc.joinSubnetAllocator.AddNetworkRange(joinSubnetCIDR, 3)
+		_ = oc.joinSubnetAllocator.AddNetworkRange(joinSubnetCIDR, 32-3)
```

Comment:
Never liked hard-coded numbers. I think the '3' should have a constant like HOST_IP_BITS so it is easier to search for.